iGCS/UserInterface/ViewControllers/GCSMapViewController.m

```diff
@@ -157,6 +165,29 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
     return YES;
 }
 
+- (void)dealloc {
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+#pragma mark - 
+#pragma mark Handle NSnotifications for UI updates
+
+// These will get called only when there is a change event
+// instead of at every tick of the telemetry data
+- (void)craftMavModeDidChange {
+    // Update armed status labels
+    if ([GCSDataManager sharedInstance].craft.isArmed) {
```

Comment:
One IF statements instead of three ternary operators!! I almost had the model return an NSAttributedstring so I could push this out of the view into the model and still get a green or red string in one go. Is that bad?